x-pack/plugins/security_solution/common/detection_engine/schemas/request/rule_schemas.test.ts

```diff
@@ -118,17 +90,12 @@ describe('create rules schema', () => {
     const decoded = createRulesSchema.decode(payload);
     const checked = exactCheck(payload, decoded);
     const message = pipe(checked, foldLeftRight);
-    expect(getPaths(left(message.errors))).toEqual([
-      'Invalid value "undefined" supplied to "risk_score"',
-      'Invalid value "undefined" supplied to "name"',
-      'Invalid value "undefined" supplied to "severity"',
-      'Invalid value "undefined" supplied to "type"',
-    ]);
+    expect(message.errors.length).toBeGreaterThan(0);
     expect(message.schema).toEqual({});
   });
 
   test('[rule_id, description, from, to, name] does not validate', () => {
-    const payload: Partial<CreateRulesSchema> = {
+    const payload = {
```

Comment:
nit: I would try to keep to utilize as many strong types as we can as it's better to fail during static analysis and/or in the IDE than during test runtime. This also gives hints to any developer coming into the file when they're adding a new test by copying/pasting a previous test and modifying they will have the type for it.

For example, with a `Partial<CreateRulesSchema>` if you misspell something when writing a test like so below you will give faster positive feedback from your IDE rather than waiting for the jest test to run. Also when people unfamiliar with our code base refactor they will get typescript compiler errors first before jest errors and those are typically easier to figure out instead of jest errors. 

<img width="589" alt="Screen Shot 2020-11-11 at 2 46 18 PM" src="https://user-images.githubusercontent.com/1151048/98868069-ccea4c00-242c-11eb-9d77-679bce9700a0.png">